app/sass/screen.scss

```diff
@@ -0,0 +1,67 @@
+@import "compass/reset";
+
+body {
+	font-family: sans-serif;
+	font-size: 14px;
+	color: #333;
+}
+h1{
+	font-size: 56px;
+}
+h2{
+	font-size: 36px;
+}
+.head, .prompt, .canvas, .timer {
+	padding: 10px;
+}
+.inline-head{
+	display: flex;
+}
+.canvas{
+	text-align: center;
+	display:block;
+	width: 95%;
+	height: 80%;
+}
+#minutes, #seconds, #colon{
```

Comment:
the style guide says not to use # id selectors... i'm a little torn on this... up to you https://github.com/airbnb/css